### PR TITLE
adding sky glitch exercise

### PR DESCRIPTION
**Pull request by afvc**

### CODE REVIEW

File: docs/primitives/a-sky.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ type: primitives
 layout: docs
 parent_section: primitives
 source_code: src/extras/primitives/primitives/a-sky.js
+examples:
+  - title: Ex1 —  Sky
```
Comment (ngokevin) [inline docs/primitives/a-sky.md]: Perhaps just call `Adding a Sky` if there's only going to be one?

File: docs/primitives/a-sky.md [hunk 2]
```diff
@@ -4,6 +4,18 @@ type: primitives
 layout: docs
 parent_section: primitives
 source_code: src/extras/primitives/primitives/a-sky.js
+examples:
```
Comment (ngokevin) [inline docs/primitives/a-sky.md]: Thanks! I might remix them to rename like `aframe-*` and then format the code a bit to match the other ones (indents, less whitespace).

File: docs/primitives/a-sky.md [hunk 3]
```diff
@@ -4,6 +4,18 @@ type: primitives
...
+  - title:Adding a Sky
+    src: https://glitch.com/edit/#!/ex-1-sky?path=index.html:1:0
+
+ - title:Videosphere Sky
+    src:https://glitch.com/edit/#!/ex-1a-videosphere
+
+ - title:Change Sky
```
Comment (ngokevin) [inline docs/primitives/a-sky.md]: I'm working on updating this, but we prescribe JS should be contained within components, not raw script tags: https://aframe.io/docs/0.6.0/introduction/javascript-events-dom-apis.html

File: docs/primitives/a-sky.md (hunk 2 above)
Comment (afvc) [reply docs/primitives/a-sky.md]: Yes you are in fact right, it's just I haven't gotten used to that structure just yet. I'm sorry for giving you extra work

**Pull request merged**